apps/fluent-tester/macos/Podfile.lock

```diff
@@ -704,4 +704,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0e740f9fca1901b692b84c05390764ca72e2fb41
 
-COCOAPODS: 1.14.2
+COCOAPODS: 1.15.0
```

Comment:
Could you revert this file from the change? cocoapods 1.15 actually isn't entirely compatible with React Native. I also usually don't update the podfile lock because it's kinda useless and causes lots of conflicts.